src/v2/repositories/implementations/SystemRepository.ts

```diff
@@ -80,10 +81,14 @@ export class SystemRepository implements ISystemRepository {
     return blockHash;
   }
 
-  public async getBlockTimeInSeconds(): Promise<number> {
-    const api = await this.api.getApi();
-    const blockTime = <u64>api.consts.aura.slotDuration;
+  public async getBlockTimeInSeconds(blockNumber?: number): Promise<number> {
+    const api = await this.api.getApi(blockNumber);
+
+    if (api.consts.aura) {
```

Comment:
is it means that the async backing enabled?

Reply:
Actually this means that `aura` pallet is not present on chain, which is the case if you ask for block time for some very old block.